README.rst

```diff
@@ -6,57 +6,138 @@ dirty_cat
+  Example: a *Sales Manager* might have a similar salary to a *Marketing Manager*,
+  as they are both managers
+
+- **Typos**: misspelled words such as *Lawyier* (instead of *Lawyer*),
+  refers to the same job, despite being two different words.
+
+- **Variations**: in some languages - such as French - a same entity
+  (such as a job title) might have different denominations:
+  *Policier* (police officer, male) and *Policière* (police officer, female).
+  Same problem as typos: we have different, very similar words designating the same job.
+
+Traditional encoding methods such as **one-hot cannot grasp these similarities**,
+and therefore would encode these values **independently**,
+which directly affects the machine learning model performance.
+
+Historically, this problem is addressed by cleaning the data manually,
+which requires a lot of work.
+
+dirty_cat encoders try to remedy this issue by providing **automatic methods
+to directly process this kind of dirty data in the machine learning pipelines**.
+They can be easily integrated in already existing pipelines as they are
+compliant with scikit-learn's API.
+
+As such, they are **drop-in replacements** for scikit-learn encoders !
+
+Those provided by dirty_cat include:
+
+- The `SimilarityEncoder <https://dirty-cat.github.io/stable/generated/dirty_cat.SimilarityEncoder.html>`_
+- The `GapEncoder <https://dirty-cat.github.io/stable/generated/dirty_cat.GapEncoder.html>`_
+- The `MinHashEncoder <https://dirty-cat.github.io/stable/generated/dirty_cat.MinHashEncoder.html>`_
+
+The library also implements other tools relevant to this issue,
+and generally useful in machine learning settings:
+
+- The `SuperVectorizer <https://dirty-cat.github.io/stable/generated/dirty_cat.SuperVectorizer.html>`_
+- The `DatetimeEncoder <https://dirty-cat.github.io/stable/generated/dirty_cat.DatetimeEncoder.html>`_
+
+You can find more details and examples on `the website <https://dirty-cat.github.io/>`_.
+
+What dirty_cat does not
+~~~~~~~~~~~~~~~~~~~~~~~
+
+`Semantic similarities <https://en.wikipedia.org/wiki/Semantic_similarity>`_
+are currently not supported.
+For example, the similarity between *car* and *automobile* is outside the reach
+of the methods implemented here.
+
+This kind of problem is tackled by
+`Natural Language Processing <https://en.wikipedia.org/wiki/Natural_language_processing>`_
+methods.
+
+dirty_cat can still help with handling typos and variations in this kind of setting.
 
 Installation
 ------------
 
+dirty_cat can be easily installed via `pip`::
+
+    pip install dirty_cat
+
 Dependencies
 ~~~~~~~~~~~~
 
-dirty_cat requires:
+Dependencies and minimal versions are listed in the file `min-requirements.txt <https://github.com/dirty-cat/dirty_cat/blob/master/requirements-min.txt>`_
 
-- Python (>= 3.8)
-- NumPy (>= 1.17.3)
-- SciPy (>= 1.4.0)
-- scikit-learn (>= 0.22.0)
-- pandas (>= 1.2.0)
+Related projects
+----------------
```

Comment:
This information should be added somewhere on the website, and not on the README. The README is not the documentation. It is less visible, and less searchable than a web page.